src/codeu/chat/relay/ServerFrontEnd.java

```diff
@@ -62,6 +63,43 @@ public void write(OutputStream out, Relay.Bundle.Component value) throws IOExcep
       Time.SERIALIZER.write(out, value.time());
     }
   };
+  
+  private static final Serializer<Relay.Bundle.ConversationComponent> CONVERSATIONCOMPONENT_SERIALIZER =
```

Comment:
I prefer an underscore between CONVERSATION and COMPONENT, since it mirror the camelcase class name